src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import core.basesyntax.db.Storage;
```

Comment:
It seems that the `Storage` import is not used in the `Main` class. Please remove any unused imports to keep the code clean.